client/common/cmdline.c

```diff
@@ -3241,6 +3255,59 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 					return COMMAND_LINE_ERROR_UNEXPECTED_VALUE;
 			}
 		}
+		CommandLineSwitchCase(arg, "pin")
```

Comment:
Moving these options to `/smartcard-logon:pin:xxx,...` might be more appropriate as they do not make sense without it?

Reply:
Same for `SmartcardPin` and `Pin`